src/components/material/ReviewExternal.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ExternalReview } from "../../core/dbc-gateway/generated/graphql";
+import ReviewHearts from "./ReviewHearts";
+import ReviewMetadata from "./ReviewMetadata";
+
+export interface ReviewExternalProps {
+  review: ExternalReview;
+}
+
+const ReviewExternal: React.FC<ReviewExternalProps> = ({ review }) => {
+  return (
+    <li className="review text-small-caption">
+      {(review.author || review.date) && (
+        <ReviewMetadata author={review.author} date={review.date} />
+      )}
+      {review.rating && <ReviewHearts amountOfHearts={review.rating} />}
+      {review.urls &&
+        review.urls.length >= 1 &&
```

Comment:
Consider if this condition is necessary. If `urls` is an empty array then it should just be `map`'ed 0 times.

Reply:
Great comment! Thanks :) I removed it